libs/template/cli/package.json

```diff
@@ -10,14 +10,15 @@
   "bin": {
     "oryx": "./bin/cli.js"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
Why do you think these are dev dependencies?

Reply:
they are not yet, but I think we should build cli bin to make it work without all the oryx deps, like `create-oryx`. Then they will be real devDeps.